config/runtime.exs

```diff
@@ -96,14 +96,15 @@ if config_env() in [:prod, :demo] do
 
   # Update catalog path to the current application dir during runtime
   config :wanda, Wanda.Catalog,
-    catalog_path:
+    catalog_paths: [
       System.get_env(
         "CATALOG_PATH",
         Application.app_dir(
           :wanda,
           "priv/catalog"
         )
       )
+    ]
```

Comment:
Meh, I would like to introduce another variable for user's checks, _or_ since we never really leveraged this we could reuse the same env var for users' checks and just put a straight path here.

Reply:
Yeah I was leaning towards having something hardcoded :+1: